puppet/modules/jenkins_job_builder/files/config/scripts/gemset_cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ruby=2.0.0
```

Comment:
Be a little careful here, the jobs are subtly different.  Can it be parameterised, or do we create multiple publishers for each variant?

1. Some jobs that use Ruby, e.g. deploy_web, just have a hardcoded version of Ruby here.  Most use Ruby version as an axis so `${ruby}` is already set.
2. test_develop etc will also call rake db:drop
3. Plugin jobs will also cd to the Foreman checkout, then db:drop
4. I've forgotten the fourth.
2.